astroquery/esa/hubble/core.py

```diff
@@ -541,7 +550,47 @@ def query_target(self, name, *, filename=None, output_format='votable',
 
         return table
 
-    def query_hst_tap(self, query, async_job=False, output_file=None,
+    def query_tap(self, query, *, async_job=False, output_file=None,
+                  output_format="votable", verbose=False):
+        """Launches a synchronous or asynchronous job to query the HST tap
+
+                Parameters
+                ----------
+                query : str, mandatory
+                    query (adql) to be executed
+                async_job : bool, optional, default 'False'
+                    executes the query (job) in asynchronous/synchronous mode (default
+                    synchronous)
+                output_file : str, optional, default None
+                    file name where the results are saved if dumpToFile is True.
+                    If this parameter is not provided, the jobid is used instead
+                output_format : str, optional, default 'votable'
+                    results format
+                verbose : bool, optional, default 'False'
+                    flag to display information about the process
+
+                Returns
+                -------
+                A table object
+                """
+        if async_job:
+            job = self._tap.launch_job_async(query=query,
+                                             output_file=output_file,
+                                             output_format=output_format,
+                                             verbose=verbose,
+                                             dump_to_file=output_file is not None)
+        else:
+            job = self._tap.launch_job(query=query, output_file=output_file,
+                                       output_format=output_format,
+                                       verbose=verbose,
+                                       dump_to_file=output_file is not None)
+        table = job.get_results()
+        return table
+
+    @deprecated(since="0.4.7", message=("Use of query_hst_tap method is no longer supported. "
+                "Please use query_tap method instead, with the same arguments."),
+                alternative="query_tap")
```

Comment:
`message` does not have to be specified. The following works:
```suggestion
    @deprecated(since="0.4.7", alternative="query_tap")
```
The resulting warning is
```
WARNING: AstropyDeprecationWarning: The query_hst_tap function is deprecated and may be removed in a future version.
        Use query_tap instead.
```

Reply:
Ok, I have removed the AstropyDeprecationWarning